### ISSUE TITLE
Buttons appearing/disappearing not very smooth

### ISSUE THREAD

**Issue by epicfaace** [codalab/codalab-worksheets#1269]
It doesn't feel smooth when the buttons appear/disappear. Possible things we can look into:
- Show buttons only after a delay, not right when the user hovers over
- Add a fade in animation
- Make sure that the cursor above any button is always a hand (note how in the animation, sometimes it's a pointer and thus clicking won't do anything unless the cursor is in the "right spot")

![anim](https://user-images.githubusercontent.com/1689183/60401887-b0ea5100-9b3c-11e9-888a-6affc3f58970.gif)

**Comment by percyliang** [comment]
I'd prefer to have the buttons show up as quickly as possible so as to not slow down the user when they're trying to add things.  So maybe we can look at the third option.

**Comment by epicfaace** [comment]
Closing this in favor of #1272 -- we will move all buttons to the navbar and not have this appearing / disappearing functionality.
